tc/sgx/trusted_worker_manager/enclave/singleton/CMakeLists.txt

```diff
@@ -65,14 +65,8 @@ LINK_ENCLAVE_COMMON_WORKLOAD_LIBRARIES()
 # ADDITIONAL LIBRARIES
 # XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
-# Add echo workload
-TARGET_LINK_LIBRARIES(${PROJECT_NAME} -Wl,--whole-archive -lecho_wo -Wl,--no-whole-archive)
-# Add heart disease eval workload
-TARGET_LINK_LIBRARIES(${PROJECT_NAME} -Wl,--whole-archive -lheart_disease_eval -Wl,--no-whole-archive)
-# Add example inside out workload
-TARGET_LINK_LIBRARIES(${PROJECT_NAME} -Wl,--whole-archive -linside_out_eval -Wl,--no-whole-archive)
-# Add simple wallet workload
-TARGET_LINK_LIBRARIES(${PROJECT_NAME} -Wl,--whole-archive -lsimple_wallet -Wl,--no-whole-archive)
+INCLUDE(../CMakeWorkloads.txt)
```

Comment:
Please add Include statement at top of the file. Move all statements in that file to macro/function and then all the required function.

Reply:
Done